VERSION

```diff
@@ -1 +1 @@
-0.1.0
+0.1.1
```

Comment:
if we aren't going to change the version when we add things we might as well just call it 1.0-SNAPSHOT the whole time